frontend/src/scenes/paths/NewPaths.js

```diff
@@ -194,22 +207,28 @@ export function NewPaths({ dashboardItemId = null, filters = null, color = 'whit
 
         link.append('path')
             .attr('d', Sankey.sankeyLinkHorizontal())
-            .attr('id', (d) => `path${d.index}`)
+            .attr('id', (d) => `path-${d.index}`)
             .attr('stroke-width', (d) => {
                 return Math.max(1, d.width)
             })
             .on('mouseover', (data) => {
```

Comment:
We should also make the cards appear when you hover a link. Right now you really have to line up your cursor with the thin bar for the card to appear.

The cards should hide again when you move the mouse away not only when you mouse over a different node

Reply:
Why does it need to be pinned? I was just noticing the same issue and wondering why it didn't collapse when leaving the path item.